.cirrus.yml

```diff
@@ -0,0 +1,21 @@
+container:
+  image: cirrusci/flutter:latest
+
+test_task:
+  # pub_cache:
+  #   folder: $HOME/.pub-cache
+  #   fingerprint_script: cat pubspec.lock
+  #   populate_script: pub get
+  pub_script: flutter packages get # later when we seperate graphql_client and graphql_flutter, we can change this
+  format_script: dartfmt **/*.dart -n --set-exit-if-changed
+  # analyze_script: dartanalyzer --fatal-infos .
+  publishable_script: pub publish --dry-run
+  environment:
+    CODECOV_TOKEN: ENCRYPTED[fa57a1dda8bca6ad45ef10043f632b68b7824b60c634415f741a847788c1a44ae9374155e3af3346fb9b6d72bac0e1d3]
```

Comment:
@truongsinh is this sensitive?

Reply:
This one has been encrypted with my repo, so it's not sensitive, but not working when merged to upstream. By not working, I mean the code coverage data is processed.